Update docs for GA MutableCSINodeAllocatableCount (#54020)
kubernetes/website: content/en/docs/reference/command-line-tools-reference/feature-gates/MutableCSINodeAllocatableCount.md

  - stage: beta
    defaultValue: true
    fromVersion: "1.35"
    toVersion: "1.35"
  - stage: stable
    defaultValue: true
    fromVersion: "1.36"
---

> When this feature gate is enabled, the `.spec.drivers[*].allocatable.count` field of a CSINode becomes mutable, and a new field, `nodeAllocatableUpdatePeriodSeconds`, is available in the CSIDriver object. This allows periodic updates to a node's reported allocatable volume capacity, preventing stateful pods from becoming stuck due to outdated information that the kube-scheduler relies on.

Reviewer: ```suggestion
Make the `.spec.drivers[*].allocatable.count` field of a CSINode mutable.
Also, enable a CSIDriver field, `nodeAllocatableUpdatePeriodSeconds`.

This allows periodic updates to a node's reported allocatable volume capacity,
preventing stateful pods from becoming stuck due to outdated information
that the kube-scheduler would otherwise rely upon.
```
